### DISCUSSION TITLE
Training of TC2-DDC model on phonemes

### DISCUSSION THREAD

**Discussion by swarajdalmia** [coqui-ai/TTS#744] [General Q&A]
I am training the TC2-DDC model on phonemes. While the alignment of the train and eval audio's looks good, there seems to be some issue with the test audios. I'm attaching the figures for all 3 below.  Does the below mean, i need to restart training and have better values for gradual training of r or is this expected and the alignment will improve.  These alignments are after 50 epochs and the current value of r at this step is 2. I am going to be further reducing to 1 after 25k more steps.

<img width="399" alt="Screenshot 2021-08-18 at 10 33 35 AM" src="https://user-images.githubusercontent.com/11361166/129839217-3a35a465-dd37-4158-8210-de7514917fc4.png">
<img width="405" alt="Screenshot 2021-08-18 at 10 42 06 AM" src="https://user-images.githubusercontent.com/11361166/129839436-daec23be-cc02-43bd-aeac-608b8176a339.png">
<img width="400" alt="Screenshot 2021-08-18 at 10 33 57 AM" src="https://user-images.githubusercontent.com/11361166/129839227-a5e8ad9d-0ec1-4a23-a588-bfc0c4e6c95e.png">

**Comment by erogol** [comment]
It looks like your stopnet does not work well. How does the train/eval stopnet loss look like?
